src/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -123,6 +123,48 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ true, { ignore: ["consecutive-duplicates-with-different-values"] } ],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "p { font-size: 16px; font-size: 1rem; }",
+  }, {
+    code: "p { display: inline-block; font-size: 16px; font-size: 1rem; }",
+  }, {
+    code: "p { font-size: 16px; font-size: 1rem; color: red; }",
+  }, {
+    code: "p { display: inline-block; font-size: 16px; font-size: 1rem; color: red; }",
+  } ],
+
+  reject: [ {
+    code: "p { font-size: 16px; font-weight: 400; font-size: 16px; }",
+    message: messages.rejected("font-size"),
+  }, {
+    code: "p { display: inline-block; font-size: 16px; font-weight: 400; font-size: 16px; }",
+    message: messages.rejected("font-size"),
+  }, {
+    code: "p { font-size: 16px; font-weight: 400; font-size: 16px; color: red; }",
+    message: messages.rejected("font-size"),
+  }, {
+    code: "p { display: inline-block; font-size: 16px; font-weight: 400; font-size: 16px; color: red; }",
+    message: messages.rejected("font-size"),
+  }, {
+    code: "p { font-size: 16px; font-size: 16px; }",
+    message: messages.rejected("16px"),
+  }, {
+    code: "p { display: inline-block; font-size: 16px; font-size: 16px; font-weight: 400; color: red; }",
+    message: messages.rejected("16px"),
+  }, {
+    code: "p { font-size: 16px; font-size: 16px; font-weight: 400; color: red; }",
+    message: messages.rejected("16px"),
+  }, {
+    code: "p { display: inline-block; font-weight: 400; font-size: 16px; font-size: 16px; }",
+    message: messages.rejected("16px"),
+  } ],
+})
```

Comment:
It seems to me that most of these tests are actually testing the same thing (consecutive declarations with the same values are flagged). Adding descriptions might help clarify the intention to make sure we're not adding unnecessary tests.